circleci: failed tests: TestCheckInconsistent

The following test appears to have failed:

[#22674](https://circleci.com/gh/cockroachdb/cockroach/22674):

```
I160913 18:16:11.559406 14736 storage/replica_command.go:1929  [s1,r1{/Min-/Max}] waited for compute checksum for 1.264951ms
I160913 18:16:11.560627 14745 storage/replica_command.go:1929  waited for compute checksum for 78.64µs
E160913 18:16:11.560779 14745 storage/stores_server.go:105  consistency check failed on range ID 1: expected checksum ea94892003096cc6e694f98214227fabb02016a4a9bb92d96ec7bc9275e30ca1845d1ededf2ef4a2442d7cf49b6c53d1ce1df1eb1ee8737409ed1bbd59fe33b1, got 11b5100e33004acdcfd3c92c393e3a3d7911651f5774fcfb2f13cc479254d6e4d996c169079e3a6184909875af52546ad1dab210726970f0480052a3a1578ea8
E160913 18:16:11.562397 14754 storage/replica_command.go:1856  [s1,r1{/Min-/Max}] replica {2 2 2} is inconsistent: expected checksum ea94892003096cc6e694f98214227fabb02016a4a9bb92d96ec7bc9275e30ca1845d1ededf2ef4a2442d7cf49b6c53d1ce1df1eb1ee8737409ed1bbd59fe33b1, got 11b5100e33004acdcfd3c92c393e3a3d7911651f5774fcfb2f13cc479254d6e4d996c169079e3a6184909875af52546ad1dab210726970f0480052a3a1578ea8
k:v = ("e" ("e"), 0.000000000,106, 000000000154) not present on lease holder
panic: sync: negative WaitGroup counter [recovered]
	panic: sync: negative WaitGroup counter

goroutine 14754 [running]:
panic(0x1afb5c0, 0xc420affd20)
	/usr/local/go/src/runtime/panic.go:500 +0x1ae
github.com/cockroachdb/cockroach/util/stop.(*Stopper).Recover(0xc4209a0120)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:176 +0xd8
panic(0x1afb5c0, 0xc420affd20)
	/usr/local/go/src/runtime/panic.go:458 +0x271
sync.(*WaitGroup).Add(0xc42056bad0, 0xffffffffffffffff)
	/usr/local/go/src/sync/waitgroup.go:75 +0x27c
sync.(*WaitGroup).Done(0xc42056bad0)
	/usr/local/go/src/sync/waitgroup.go:100 +0x42
github.com/cockroachdb/cockroach/storage.(*Replica).CheckConsistency.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/replica_command.go:1857 +0xca9
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunAsyncTask.func1(0xc4209a0120, 0x2587e93, 0x1a, 0x741, 0xc420ac2000)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:231 +0xa3
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunAsyncTask
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:232 +0xca

goroutine 1 [chan receive]:
testing.(*T).Run(0xc42024d740, 0x1cb097b, 0x15, 0x1f72190, 0x9a2e01)
	/usr/local/go/src/testing/testing.go:647 +0x56e
testing.RunTests.func1(0xc42024d740)
	/usr/local/go/src/testing/testing.go:793 +0xba
testing.tRunner(0xc42024d740, 0xc420251b38)
	/usr/local/go/src/testing/testing.go:610 +0xca
testing.RunTests(0x1f73600, 0x28807e0, 0x143, 0x143, 0xc4202494c8)
	/usr/local/go/src/testing/testing.go:799 +0x4bb
testing.(*M).Run(0xc420251ef0, 0xc420281ce0)
	/usr/local/go/src/testing/testing.go:743 +0x130
github.com/cockroachdb/cockroach/storage_test.TestMain(0xc420251ef0)
	/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:57 +0x287
main.main()
	github.com/cockroachdb/cockroach/storage/_test/_testmain.go:708 +0x1b6

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1

goroutine 14590 [select]:
google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc4211c3900, 0xc421175970, 0x5, 0x5, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/transport/transport.go:141 +0x89c
google.golang.org/grpc/transport.(*Stream).Read(0xc42085c000, 0xc421175970, 0x5, 0x5, 0xc420409ae0, 0xc420409ae0, 0x96584f)
	/go/src/google.golang.org/grpc/transport/transport.go:314 +0x79
io.ReadAtLeast(0x288a740, 0xc42085c000, 0xc421175970, 0x5, 0x5, 0x5, 0x60f20f, 0xc420409c3c, 0x0)
	/usr/local/go/src/io/io.go:307 +0xb2
io.ReadFull(0x288a740, 0xc42085c000, 0xc421175970, 0x5, 0x5, 0xc420fa2d78, 0xc420409c3c, 0x5db700)
	/usr/local/go/src/io/io.go:325 +0x73
google.golang.org/grpc.(*parser).recvMsg(0xc421175960, 0x7fffffff, 0xc4211c3a40, 0xc420409c50, 0x62cfc0, 0xc4211c3b00, 0xc420409e80, 0xc1cfd3)
	/go/src/google.golang.org/grpc/rpc_util.go:231 +0xa4
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc4211c3ac0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 20 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x2bca300)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1032 +0x85
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:613 +0xc3

goroutine 14546 [select]:
github.com/cockroachdb/cockroach/rpc.(*Context).runHeartbeat(0xc421068000, 0xc42083af00, 0xc420eff5b0, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:294 +0x813
github.com/cockroachdb/cockroach/rpc.(*Context).GRPCDial.func1.1.1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:217 +0xb3
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc420fa3d70)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14597 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).bootstrap.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:911 +0x621
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc421176200)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14614 [select]:
github.com/cockroachdb/cockroach/storage.NewRaftTransport.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:182 +0x12f6
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc421176930)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14544 [IO wait]:
net.runtime_pollWait(0x7fdf9ea6eab8, 0x72, 0x5e4823)
	/usr/local/go/src/runtime/netpoll.go:160 +0x5e
net.(*pollDesc).wait(0xc421153560, 0x72, 0x5e10d8, 0x1b72820)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x5b
net.(*pollDesc).waitRead(0xc421153560, 0x288e300, 0xc4200120a8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x42
net.(*netFD).accept(0xc421153500, 0x0, 0x288b640, 0xc420e801a0)
	/usr/local/go/src/net/fd_unix.go:419 +0x2b8
net.(*TCPListener).accept(0xc4201184b8, 0xc420028020, 0xc420037e98, 0x60959f)
	/usr/local/go/src/net/tcpsock_posix.go:132 +0x51
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc420aa5ac0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14621 [IO wait]:
net.runtime_pollWait(0x7fdf9ea6ef38, 0x72, 0x5e4823)
	/usr/local/go/src/runtime/netpoll.go:160 +0x5e
net.(*pollDesc).wait(0xc420adc840, 0x72, 0x5e10d8, 0x1b72820)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x5b
net.(*pollDesc).waitRead(0xc420adc840, 0x288e300, 0xc4200120a8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x42
net.(*netFD).Read(0xc420adc7e0, 0xc4204d6000, 0x8000, 0x8000, 0x0, 0x288e300, 0xc4200120a8)
	/usr/local/go/src/net/fd_unix.go:243 +0x1d9
net.(*conn).Read(0xc420118530, 0xc4204d6000, 0x8000, 0x8000, 0x3e, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x97
--
google.golang.org/grpc.(*Server).handleRawConn(0xc420149400, 0x28a0de0, 0xc420118530)
	/go/src/google.golang.org/grpc/server.go:388 +0x5c3
created by google.golang.org/grpc.(*Server).Serve
	/go/src/google.golang.org/grpc/server.go:357 +0x20a

goroutine 14582 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:340 +0x519
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0510, 0xc421174f00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14560 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1066 +0x342
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc420effce0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14576 [IO wait]:
net.runtime_pollWait(0x7fdf9ea6edb8, 0x72, 0x5e4823)
	/usr/local/go/src/runtime/netpoll.go:160 +0x5e
net.(*pollDesc).wait(0xc420f02450, 0x72, 0x5e10d8, 0x1b72820)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x5b
net.(*pollDesc).waitRead(0xc420f02450, 0x288e300, 0xc4200120a8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x42
net.(*netFD).Read(0xc420f023f0, 0xc420044000, 0x8000, 0x8000, 0x0, 0x288e300, 0xc4200120a8)
	/usr/local/go/src/net/fd_unix.go:243 +0x1d9
net.(*conn).Read(0xc4201181a8, 0xc420044000, 0x8000, 0x8000, 0xc4211f8f08, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x97
--
google.golang.org/grpc/transport.(*http2Client).reader(0xc4209a6200)
	/go/src/google.golang.org/grpc/transport/http2_client.go:921 +0x154
created by google.golang.org/grpc/transport.newHTTP2Client
	/go/src/google.golang.org/grpc/transport/http2_client.go:203 +0xc1a

goroutine 14648 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc420adc230, 0xc4209a0900)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x550
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x4b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0900, 0xc4208100e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14711 [select]:
github.com/cockroachdb/cockroach/storage.(*RaftTransport).processQueue(0xc4211fa140, 0x7fdf00000002, 0xc4204393e0, 0xc420f6b580, 0x28a0cc0, 0xc421073c30, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:449 +0x630
github.com/cockroachdb/cockroach/storage.(*RaftTransport).connectAndProcess.func1(0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:397 +0x4f2
github.com/rubyist/circuitbreaker.(*Breaker).Call(0xc420857170, 0xc420d9d040, 0x0, 0x1, 0xc42000cba0)
	/go/src/github.com/rubyist/circuitbreaker/circuitbreaker.go:339 +0x7c
github.com/cockroachdb/cockroach/storage.(*RaftTransport).connectAndProcess(0xc4211fa140, 0xc400000002, 0xc4204393e0, 0xc420f6b580)
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:398 +0x204
github.com/cockroachdb/cockroach/storage.(*RaftTransport).SendAsync.func2.1()
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:528 +0x64
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc421055770)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14631 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc420240e00, 0xecf6a396b, 0xc419108cdf, 0x2bc91a0, 0xc420a7c0c0, 0xc4209a0510, 0x0, 0xc420a7ca40)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:201 +0x299
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:264 +0x540
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0510, 0xc420aa4b40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14701 [select]:
google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc420d9d640, 0xc42106b210, 0x5, 0x5, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/transport/transport.go:141 +0x89c
google.golang.org/grpc/transport.(*Stream).Read(0xc4200c0870, 0xc42106b210, 0x5, 0x5, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/transport/transport.go:314 +0x79
io.ReadAtLeast(0x288a740, 0xc4200c0870, 0xc42106b210, 0x5, 0x5, 0x5, 0x0, 0x0, 0x5e3256)
	/usr/local/go/src/io/io.go:307 +0xb2
io.ReadFull(0x288a740, 0xc4200c0870, 0xc42106b210, 0x5, 0x5, 0x60000000000, 0x0, 0x48)
	/usr/local/go/src/io/io.go:325 +0x73
google.golang.org/grpc.(*parser).recvMsg(0xc42106b200, 0x7fffffff, 0xc4210ad498, 0xc400000001, 0x0, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/rpc_util.go:231 +0xa4
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc420fd16e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14726 [select]:
github.com/cockroachdb/cockroach/storage.(*RaftTransport).processQueue(0xc4211540a0, 0xc400000001, 0xc4208e4720, 0xc420d9c280, 0x28a0cc0, 0xc420e0ce70, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:449 +0x630
github.com/cockroachdb/cockroach/storage.(*RaftTransport).connectAndProcess.func1(0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:397 +0x4f2
github.com/rubyist/circuitbreaker.(*Breaker).Call(0xc420245e60, 0xc420b2f5c0, 0x0, 0x1, 0xc4210ad440)
	/go/src/github.com/rubyist/circuitbreaker/circuitbreaker.go:339 +0x7c
github.com/cockroachdb/cockroach/storage.(*RaftTransport).connectAndProcess(0xc4211540a0, 0xc400000001, 0xc4208e4720, 0xc420d9c280)
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:398 +0x204
github.com/cockroachdb/cockroach/storage.(*RaftTransport).SendAsync.func2.1()
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:528 +0x64
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc420db2720)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14592 [select]:
google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc4211c3dc0, 0xc421175c30, 0x5, 0x5, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/transport/transport.go:141 +0x89c
google.golang.org/grpc/transport.(*Stream).Read(0xc42085c1e0, 0xc421175c30, 0x5, 0x5, 0x0, 0x636fe2, 0x60f20f)
	/go/src/google.golang.org/grpc/transport/transport.go:314 +0x79
io.ReadAtLeast(0x288a740, 0xc42085c1e0, 0xc421175c30, 0x5, 0x5, 0x5, 0xbd63d1, 0x24, 0xc4210438c0)
	/usr/local/go/src/io/io.go:307 +0xb2
io.ReadFull(0x288a740, 0xc42085c1e0, 0xc421175c30, 0x5, 0x5, 0x2, 0xc420eff7d8, 0x5ff615)
	/usr/local/go/src/io/io.go:325 +0x73
google.golang.org/grpc.(*parser).recvMsg(0xc421175c20, 0x7fffffff, 0x5d7455, 0xc4210a9618, 0x0, 0x0, 0x6, 0x0)
	/go/src/google.golang.org/grpc/rpc_util.go:231 +0xa4
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc420a7c040)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14559 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1047 +0x342
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc420effcd0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14591 [select]:
github.com/cockroachdb/cockroach/gossip.(*server).Gossip(0xc420130460, 0x28a0360, 0xc420efe5a0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/gossip/server.go:180 +0x73e
github.com/cockroachdb/cockroach/gossip._Gossip_Gossip_Handler(0x1bf4ca0, 0xc420130460, 0x289c9a0, 0xc420856630, 0xc421175970, 0x1)
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.pb.go:209 +0xda
google.golang.org/grpc.(*Server).processStreamingRPC(0xc420148000, 0x28a08a0, 0xc420856000, 0xc42085c1e0, 0xc420fa2cf0, 0x286f5e0, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/server.go:689 +0xc7d
google.golang.org/grpc.(*Server).handleStream(0xc420148000, 0x28a08a0, 0xc420856000, 0xc42085c1e0, 0x0)
	/go/src/google.golang.org/grpc/server.go:773 +0x170e
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc4211760c0, 0xc420148000, 0x28a08a0, 0xc420856000, 0xc42085c1e0)
	/go/src/google.golang.org/grpc/server.go:422 +0xb9
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/google.golang.org/grpc/server.go:423 +0xbb

goroutine 14634 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:418 +0x395
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0510, 0xc420aa4ba0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14702 [select]:
github.com/cockroachdb/cockroach/storage.(*RaftTransport).RaftMessage(0xc4211fa140, 0x28a0660, 0xc421073fd0, 0xc4208d2ba8, 0xc4208d2bb8)
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:315 +0x305
github.com/cockroachdb/cockroach/storage._MultiRaft_RaftMessage_Handler(0x1c2e9c0, 0xc4211fa140, 0x289c9a0, 0xc420244d80, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/raft.pb.go:167 +0xda
google.golang.org/grpc.(*Server).processStreamingRPC(0xc420148000, 0x28a08a0, 0xc420856000, 0xc4200c0960, 0xc420fa2450, 0x286f880, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/server.go:689 +0xc7d
google.golang.org/grpc.(*Server).handleStream(0xc420148000, 0x28a08a0, 0xc420856000, 0xc4200c0960, 0x0)
	/go/src/google.golang.org/grpc/server.go:773 +0x170e
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc4211760c0, 0xc420148000, 0x28a08a0, 0xc420856000, 0xc4200c0960)
	/go/src/google.golang.org/grpc/server.go:422 +0xb9
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/google.golang.org/grpc/server.go:423 +0xbb

goroutine 14727 [select]:
google.golang.org/grpc.NewClientStream.func2(0x28a0840, 0xc4209a6000, 0xc4209a2c30, 0xc4201ce240)
	/go/src/google.golang.org/grpc/stream.go:197 +0x54e
created by google.golang.org/grpc.NewClientStream
	/go/src/google.golang.org/grpc/stream.go:217 +0x10a5

goroutine 14604 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2628 +0x482
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0510, 0xc4211763c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14541 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:340 +0x519
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc420aa4c60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14700 [select]:
google.golang.org/grpc.NewClientStream.func2(0x28a0840, 0xc4209a6000, 0xc4200c0870, 0xc420137c20)
	/go/src/google.golang.org/grpc/stream.go:197 +0x54e
created by google.golang.org/grpc.NewClientStream
	/go/src/google.golang.org/grpc/stream.go:217 +0x10a5

goroutine 14630 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:418 +0x395
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0510, 0xc420aa4ae0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14647 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc420adc230, 0xc4209a0900)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x550
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x4b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0900, 0xc4208100c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14555 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2628 +0x482
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc420effc60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14542 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc4210c80b0)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:208 +0xde7
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:193 +0x42
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc420efe820)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14615 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:340 +0x519
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0900, 0xc420e81380)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14580 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc4210bf3e0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:390 +0x64
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:391 +0xb61

goroutine 14558 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:972 +0x19d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc421174220)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14540 [select]:
github.com/cockroachdb/cockroach/storage.NewRaftTransport.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:182 +0x12f6
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc420efe1c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14566 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:418 +0x395
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc420e81600)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14522 [select]:
google.golang.org/grpc.(*addrConn).transportMonitor(0xc4202d4000)
	/go/src/google.golang.org/grpc/clientconn.go:720 +0x7d7
created by google.golang.org/grpc.(*ClientConn).resetAddrConn
	/go/src/google.golang.org/grpc/clientconn.go:467 +0x9c9

goroutine 14633 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:418 +0x395
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0510, 0xc420aa4b80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14685 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc420efce00, 0xecf6a396b, 0xc41a4f0287, 0x2bc91a0, 0xc420a7c0c0, 0xc4209a0900, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:201 +0x299
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:264 +0x540
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0900, 0xc420aa5fa0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14556 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).raftSnapshotStatusLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2658 +0x2d4
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc420effc70)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14654 [select]:
github.com/cockroachdb/cockroach/gossip.(*client).gossip(0xc4201310a0, 0x7fdf9ea1b9a0, 0xc4210abf80, 0xc42054d200, 0x28a0c00, 0xc421177bb0, 0xc4209a0000, 0xc421177b70, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/gossip/client.go:299 +0x4b5
github.com/cockroachdb/cockroach/gossip.(*client).start.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/client.go:126 +0x72d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc4210abf00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14738 [select]:
google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc420360700, 0xc420fb6950, 0x5, 0x5, 0x0, 0x0, 0x0)
	/go/src/google.golang.org/grpc/transport/transport.go:141 +0x89c
google.golang.org/grpc/transport.(*Stream).Read(0xc42060a960, 0xc420fb6950, 0x5, 0x5, 0x10, 0xc420a83ae8, 0x0)
	/go/src/google.golang.org/grpc/transport/transport.go:314 +0x79
io.ReadAtLeast(0x288a740, 0xc42060a960, 0xc420fb6950, 0x5, 0x5, 0x5, 0x2bd3138, 0x7fdf9ea38948, 0x22f)
	/usr/local/go/src/io/io.go:307 +0xb2
io.ReadFull(0x288a740, 0xc42060a960, 0xc420fb6950, 0x5, 0x5, 0x5e1eda, 0x2bca460, 0xc420358b60)
	/usr/local/go/src/io/io.go:325 +0x73
google.golang.org/grpc.(*parser).recvMsg(0xc420fb6940, 0x7fffffff, 0x1b67d40, 0x1c60640, 0x5e6b01, 0x7fdf9ea1bcc0, 0xb13555, 0x9be880)
	/go/src/google.golang.org/grpc/rpc_util.go:231 +0xa4
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0000, 0xc420e0b980)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14551 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc421153180, 0xc4209a0120)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x550
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x4b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4209a0120, 0xc4211740c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:187 +0x8b
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x74

goroutine 14679 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func
```
Please assign, take a look and update the issue accordingly.

--- a/storage/replica_command.go
+++ b/storage/replica_command.go
@@ -1806,6 +1806,7 @@ func (r *Replica) CheckConsistency(
 		if replica == localReplica {
 			continue
 		}
+		wg.Add(1)
 		replica := replica // per-iteration copy
 		if err := r.store.Stopper().RunAsyncTask(func() {
 			defer wg.Done()
@@ -1856,9 +1857,8 @@ func (r *Replica) CheckConsistency(
 			log.Errorf(ctx, buf.String())
 		}); err != nil {
 			log.Error(ctx, errors.Wrap(err, "could not run async CollectChecksum"))
-			continue
+			wg.Done()
 		}
-		wg.Add(1)
 	}
 	wg.Wait()
 

